x-pack/platform/plugins/shared/fleet/server/types/rest_spec/fleet_server_policy_config.ts

```diff
@@ -33,12 +69,14 @@ export const GetOneFleetServerHostRequestSchema = {
 
 export const PutFleetServerHostRequestSchema = {
   params: schema.object({ itemId: schema.string() }),
-  body: schema.object({
+  body: FleetServerHostSchema.extends({
     name: schema.maybe(schema.string()),
     host_urls: schema.maybe(schema.arrayOf(schema.string(), { minSize: 1 })),
     is_default: schema.maybe(schema.boolean({ defaultValue: false })),
     is_internal: schema.maybe(schema.boolean()),
     proxy_id: schema.nullable(schema.string()),
+    // remove is_preconfigured from schema
+    is_preconfigured: undefined,
```

Comment:
nit: could the fields without `is_preconfigured` moved to a base schema instead?